recipes/openfoam/meta.yaml

```
@@ -0,0 +1,72 @@
+{% set name = "openfoam" %}
+{% set version = "v2412" %}
```

```suggestion
{% set version = "2412" %}
```
Don't include the v in the conda version.

---

Removed the "vé as suggested